scripts/hardfork/build-and-test.sh

```diff
@@ -52,9 +52,13 @@ fi
 INIT_DIR="$PWD"
 SCRIPT_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
 
+# NOTE: Setting timezone, so if this is running in CI under NixOS, we won't 
+# panic because `/etc/localtime` being non-existent. For more, see:
+# - https://github.com/janestreet/core/blob/5a0b30cbfeb75dfc6f8d70dcceb499b250133bac/core/src/timezone_intf.ml#L27
+export TZ="UTC"
+
 if [[ $# -gt 0 ]]; then
   # Branch is specified, this is a CI run
-  ln -sf /usr/share/zoneinfo/UTC /etc/localtime
```

Comment:
So indeed this linking is not working somehow after the rewrite. Hence the fix. 

Not sure why though.

Reply:
By which I mean `/usr/share` as a link doesn't exist.